src/collision.cpp

```diff
@@ -527,6 +527,10 @@ collisionMoveResult collisionMoveSimple(Environment *env, IGameDef *gamedef,
 				is_collision = false;
 
 			CollisionInfo info;
+			if (is_collision) {
+				info.new_pos = *pos_f/10;
```

Comment:
should be `*pos_f / BS`
this can also be moved into the `if (is_collision)` on line 569

Reply:
Updated!